libresapi/src/api/TransfersHandler.h

```diff
@@ -26,6 +28,7 @@ class TransfersHandler: public ResourceRouter, Tickable
     StateTokenServer* mStateTokenServer;
     RsFiles* mFiles;
 	RsPeers* mRsPeers;
+	RsNotify* mNotify;

```

Comment:
It is preferable to use a reference instead of a pointer if possible (aka. if it is always valid and doesn't change between initialization and destruction)

Reply:
Pointer is internally used to access RsNotify object, hence there is no need to dereference it adding just another layer of complexity without any reasonable advantage

Reply:
I don't suggest to dereference it, I suggest to use a reference like +RsNotify& mNotify;+ instead of a pointer, it doesn't add complexity nor copies but has better guaranties (it can't be NULL or change in it's lifetime)